### ISSUE TITLE
Merge with go-libp2p-routing-helpers

### ISSUE THREAD

**Issue by Stebalien** [ipfs/go-ipfs-routing#11]
1. This should be a libp2p package.
2. I created go-libp2p-routing-helpers because this package wasn't in the libp2p repo.

**Comment by magik6k** [comment]
I'll handle that after searchValue stuff is done

**Comment by hacdias** [comment]
This repository is no longer maintained and has been copied over to [Boxo](https://github.com/ipfs/boxo).  In an effort to avoid noise and crippling in the Boxo repo from the weight of issues of the past, we are closing most issues and PRs in this repo.  Please feel free to open a new issue in Boxo (and reference this issue) if resolving this issue is still critical for unblocking or improving your usecase.

You can learn more in the [FAQs for the Boxo repo copying/consolidation effort](https://github.com/ipfs/boxo/wiki/Copied-or-Migrated-Repos-FAQ).
